src/plugins/util/web.js

```diff
@@ -69,6 +69,10 @@ const web = {
     wrapEnd(req)
     wrapEvents(req)
 
+    if (process.env.SIGNALFX_SERVER_TIMING_CONTEXT === 'true') {
+      res.setHeader('Server-Timing', traceParentHeader(span.context()))
```

Comment:
`web.instrument()` is used by most (all?) server and framework instrumentations. You likely want to confirm this hasn't already been set via a flag on the response.

Reply:
Right, for reentrancy?  Good thought, will do.